### PR TITLE
Sandbox owner permission adjustment

### PR DESCRIPTION
**Pull request by ssmiller25**
Implement fixes outlined in #1

### CODE REVIEW

**Comment by ssmiller25** [comment]
It compiles, but still need to run unit and integration tests (windows...)  Planning to run later this morning and make any necessary adjustments.

**Comment by ssmiller25** [comment]
...or the pipeline will just do it for me :-)

File: README.md [hunk 1]
```diff
@@ -139,15 +139,16 @@ Assuming the name of the created Sandbox is named `foo`, the following resources
 
 ### Namespace (sandbox-foo)
 
-### ClusterRole (sandbox-foo-deleter)
+### ClusterRole (sandbox-foo-admin)
 
 |Verbs|API Groups|Resources|ResourceNames|
 |---|---|---|---|
 |delete|operators.plex.dev|sandboxes|sandbox-foo|
+|patch|operators.plex.dev|sandboxes|sandbox-foo|
```
Comment (jpreese) [inline README.md]: nit: At a minimum I think we should include the `update` verb here as well (on the same row). Though to tell the truth, I can't think of a reason not to just allow all verbs on this resource.

File: deploy/user-default-role.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: sandbox-users
+  name: sandbox-user
```
Comment (jpreese) [inline deploy/user-default-role.yaml:4]: nit: We'll want to update the documentation to reflect this change as well

**Review by jpreese — COMMENTED** [review]
Thanks for putting this together, @ssmiller25! Couple small nits

File: README.md (hunk 1 above)
Comment (ssmiller25) [reply README.md]: True...maybe I'll update to give allow all verbs for that clusterrole.  I'll commit the updates and see what you think.

File: deploy/user-default-role.yaml (hunk 2 above)
Comment (ssmiller25) [reply deploy/user-default-role.yaml:4]: Documentation should be updated

File: README.md [hunk 3]
```diff
@@ -139,19 +139,20 @@ Assuming the name of the created Sandbox is named `foo`, the following resources
 
 ### Namespace (sandbox-foo)
 
-### ClusterRole (sandbox-foo-deleter)
+### ClusterRole (sandbox-foo-admins)
```
Comment (jpreese) [inline README.md]: nit: It looks like we flipped all of the Role/RoleBinding pluralities. The backing code looks correct.

**Comment by jpreese** [comment]
@ssmiller25 Another small nit, and it looks like we still need to update the plurality of the `sandbox-user` / `sandbox-users` ClusterRole/Binding combo in the documentation. YAML itself looks correct!

File: README.md (hunk 3 above)
Comment (ssmiller25) [reply README.md]: Nope, good for our docs to match our code.  Someday we'll have automated test for docs :-)

**Comment by ssmiller25** [comment]
I think the docs are updated, but let me know if you see anything else.  Thanks for looking this over.

**Comment by jpreese** [comment]
@ssmiller25 few more fixes:

It looks like we still need to:

change the sandbox-users `ClusterRole` to sandbox-user in the documentation and
change the sandbox-user `ClusterRoleBinding` to sandbox-users in the documentation

The sandbox-[name]-owner role and rolebinding looks to be changed to `user`. I think we should keep this as `owner` to be consistent with the CRD label and be more reflective of the level of permissions they have. As it stands now, the documentation doesn't match the operator code because the operator still creates `-owner` references.

**Comment by ssmiller25** [comment]
Makes sense.  Updates to docs have bee pushed.

**Comment by jpreese** [comment]
LGTM 👍

**Pull request merged**